### ISSUE TITLE
断句处基本没有停顿的处理方法

### ISSUE THREAD

**Issue by sah1234567** [index-tts/index-tts#149]
在infer.py的以下位置进行修改：
                wavs.append(wav.cpu())  # to cpu before saving
                # 在每个句子后添加0.3秒静音（采样率24000，所以0.3秒=7200个采样点）
                silence = torch.zeros((1, 7200), dtype=torch.int16)
                wavs.append(silence)
        end_time = time.perf_counter()

        # 拼接时跳过最后一个静音（避免音频末尾有多余静音）
        wav = torch.cat(wavs[:-1], dim=1) if len(wavs) > 1 else wavs[0]

**Comment by boostpapa** [comment]
建议加一个标点，会有比较明显的停顿

**Comment by index-tts** [comment]
建议在句尾加上一个标点，会有明显的停顿。

**Comment by sah1234567** [comment]
> 建议在句尾加上一个标点，会有明显的停顿。

如果让它读比较长的文章，文章切分后生成的wavs之间会出现停顿非常短暂的情况。
